app/components/trainees/record_actions/view.rb

```diff
@@ -22,6 +22,37 @@ def allow_defer?
       def allow_withdraw?
         allow_defer? || trainee.deferred?
       end
+
+      def defer_and_withdraw_links
+        case display_actions?
```

Comment:
This code confuses me a bit - is a case statement on `display_actions?` correct?

`display_actions?` returns a boolean, so it's strange to see three `when` statements following it, especially when some of those `when` statements reference methods that are called within `display_actions?` itself.

Maybe the logic does work out , but I find it a bit confusing to follow!

Is it actually an `if / elsif` statement?

```
def defer_and_withdraw_links
  if choose_both_actions?
    defer_path + " or " + withdraw_path
  elsif allow_defer?
    defer_path
  elsif allow_withdraw?
    reinstate_path + " or " + withdraw_path
  else
    ""
  end
end
```